sgp4/tests.py

```diff
@@ -85,6 +86,44 @@ def test_satrec_attributes(self):
         self.assertEqual(sat.mo, 0.3373093125574321)
         self.assertEqual(sat.no_kozai, 0.04722944544077857)
 
+    def test_satrec_sgp4init_attributes(self):
+        # Make sure the Satrec has the same attributes if initialized via sgp4init() directly
+        satnum = 5
+        jdsatepoch = 2451723.28495062
+        bstar = 2.8098e-05
+        ndot = 6.96919666594958e-13
+        nddot = 0.0
+        ecco = 0.1859667
+        argpo = 5.790416027488515
+        inclo = 0.5980929187319208
+        mo = 0.3373093125574321
+        no_kozai = 0.04722944544077857
+        nodeo = 6.08638547138321
+
+        # Make sure the Satrec ininitalized via sgp4init() has the same attributes.
+        sat = self.build_satrec_sgp4init(satnum, jdsatepoch, bstar, ndot, nddot,
+                          ecco, argpo, inclo, mo, no_kozai, nodeo)
```

Comment:
Try tweaking the indentation to be some variety of PEP-8 (at least the way GitHub is formatting it, the indent level does not match anything else in the file?). This comment also applies several places below, so try double-checking all the indentation. (If your editor doesn't do it automatically you might check if its settings are off?)

Reply:
Fixed indentation to align with open parenthesis of function on argument linewrap.